.github/workflows/lib-injection.yml

```diff
@@ -125,3 +125,52 @@ jobs:
         if: success() || failure()
         run: |
           docker compose logs
+
+  test_unit_no_instrumentation:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        variant: [
+          'dd-lib-python-init-test-django-unsupported-python',
+          'dd-lib-python-init-test-django-unsupported-package',
+        ]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build and run the app
+        run: |
+          SRC="$(pwd)"
+          cd lib-injection
+          export DDTRACE_PYTHON_VERSION="v2.6.3"
```

Comment:
I noticed this the other day for the existing tests, is it a problem we are testing with a static version of the tracer and not the current commit?

Reply:
It doesn't seem ideal. If I remember, this test design aimed to exercise installation from PyPI, limiting it to released versions.